encryption_works.md

```diff
@@ -339,53 +339,43 @@ Snowden was right to be cautious and to insist that he check Gellman's new PGP f
 
-### PGP and Email in Tails
+It's a good idea to install Tails on a fresh storage device (either a DVD or a USB stick) to ensure everything works as it should. Then, follow the instructions on [tails.boum.org](https://tails.boum.org/download/index.en.html)'s website. It's an involved process that involves knowledge of the command-line, but it's worth sticking it out. This is the hardest crypto tool to install, but it is currently the gold standard for anonymously using a  computer, particularly when coupled with PGP-encrypted e-mail with Claws Mail, the e-mail application bundled with Tails. For set-up instructions, check out [Claws](http://www.claws-mail.org/documentation.php?section=general) documentation.
 
-I discussed using Thunderbird with the Enigmail add-on to access your email and use PGP, however this software doesn't come with Tails. Tails comes with [Claws Mail](http://www.claws-mail.org/) which includes a PGP plugin.
+If a malicious attacker manages to modify or compromise your Tails USB stick, the security of the operating system can be compromised. For this reason, you should keep your USB stick on you at all times. 
 
-![](https://raw.github.com/micahflee/encryption-works/master/images/claws_mail.png)
+Had CIA Director and retired four-star general David Petraeus and his biographer Paula Broadwell decided to use Tails, Tor, OTR, and PGP, their [extramarital affair](https://en.wikipedia.org/wiki/Petraeus_scandal) likely would have remained secret.
 
-Instead of using Enigmail's PGP key management graphical user interface to import, export, generate, view details about, and sign keys, you can click on the clipboard icon in the top right of the screen and choose Manage Keys to open [Seahorse](https://live.gnome.org/Seahorse), which provides these same features.
+## Contextual Identities
 
-![](https://raw.github.com/micahflee/encryption-works/master/images/tails_manage_keys.png)
+Tails gives you a lot of technical security, but it has to be supplemented by behavioral changes on the user's part. For instance, you should use different Tails sessions for different tasks. (A session refers to a time spent using Tails, and it ends when you shut down and Tails resets. If you use Tails over two days and shut down in between, you're said to have two sessions.) 
 
-### Workflow
+If you have an anonymous e-mail address you use to communicate with a source, it's inadvisable to also log into your personal or work.
 
-To get started having private communications with your friends and colleagues with very high endpoint security, here are the steps you need to take.
+## Persistence
 
-* Meet up with your friends face-to-face. Each person should bring their own laptop and USB stick.
```

Comment:
@trevortimm It's part of a larger comment on key signing, which I moved to the PGP section. Do you still want it here?